.gitignore

```diff
@@ -1,3 +1,4 @@
 data_climate_foresight.tar
 data/ 
-*.ipynb
\ No newline at end of file
+*.ipynb
```

Comment:
Please in future submit those kind of changes as separate PR, no need to mix it.